bin/dustc

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
+    watchr = require('watchr'),
```

Comment:
I notice that it takes 1-2 seconds to recompile currently, and I see that chokidar says:

> Other node.js watching libraries:
>
>    Are not using ultra-fast non-polling fsevents watcher implementation on OS X

I might try swapping them out. You're right that chokidar is more popular according to npm.